### PR TITLE
Update README.md for new release

### PR DESCRIPTION
**Pull request by oscarbatori**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,17 +72,143 @@ $ dolt sql -q 'insert into state_populations (state, population) values
...
+
+should show something like this:
+
+    origin https://doltremoteapi.dolthub.com/org/repo
+
+If you've created the repository on the web and added a remote, you should be able to push the branch "master" to the remote named "origin" like so
+
+    dolt push origin master
+
+Once that is succeeded others can clone the repository (assuming you've given them permission.)
```
Comment (mjesuele) [inline README.md]:
```suggestion
Once that is succeeded others can clone the repository (assuming you've given them permission):
```

File: README.md [hunk 2]
```diff
@@ -105,17 +72,143 @@ $ dolt sql -q 'insert into state_populations (state, population) values
...
+
+    dolt clone org/repo
+
+
+## Other remotes
+
+dolt also supports directory, aws, and gcs based remotes:
```
Comment (mjesuele) [inline README.md]:
```suggestion
Dolt also supports directory, AWS, and GCS based remotes:
```

File: README.md [hunk 3]
```diff
@@ -105,17 +72,143 @@ $ dolt sql -q 'insert into state_populations (state, population) values
...
+
+  * file - you can use a directory as a remote that can be pushed to, cloned, and purlled from just like any other remote by providing a file uri for the directory
```
Comment (mjesuele) [inline README.md]:
```suggestion
  * file - you can use a directory as a remote that can be pushed to, cloned, and pulled from just like any other remote by providing a file URI for the directory
```

File: README.md [hunk 4]
```diff
@@ -105,17 +72,143 @@ $ dolt sql -q 'insert into state_populations (state, population) values
...
+
+    dolt remote add <remote> file:///Users/xyz/abs/path/
+
+  * aws - you can use your aws cloud resources directly (If you are interested in trying this contact the dolt team directly as we are still writing documentation).
```
Comment (mjesuele) [inline README.md]:
```suggestion
  * AWS - you can use your AWS cloud resources directly (If you are interested in trying this contact the Dolt team directly as we are still writing documentation).
```

File: README.md [hunk 5]
```diff
@@ -105,17 +72,143 @@ $ dolt sql -q 'insert into state_populations (state, population) values
...
+
+    dolt remote add <remote> aws://dynamo-table:s3-bucket/database
+
+  * gs - you can use a GCS bucket as well.  Setup here should be straight forward.  You'll need to create a gcs bucket, and you'll need to use the gcloud auth login command in order to setup your credentials.
```
Comment (mjesuele) [inline README.md]:
```suggestion
  * GCS - you can use a GCS bucket as well.  Setup here should be straight forward.  You'll need to create a GCS bucket, and you'll need to use the gcloud auth login command in order to set up your credentials.
```

File: README.md [hunk 6]
```diff
@@ -105,17 +72,143 @@ $ dolt sql -q 'insert into state_populations (state, population) values
...
+
+    dolt remote add <remote> gs://gcs-bucket/database
+
+## Issues
+If you have any issues with `Dolt`, find any bugs, or simply have a question, feel free to file an issue!
```
Comment (mjesuele) [inline README.md]:
```suggestion
If you have any issues with Dolt, find any bugs, or simply have a question, feel free to file an issue!
```

**Comment by bheni** [comment]
The README.md that we put out with the binaries is not the same README.md file that is the main repo readme.

**Comment by bheni** [comment]
Reopened after further discussion and clarification.

**Pull request merged**